storehaus-http/src/main/scala-2.10/com/twitter/storehaus/http/compat/package.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2016 Twitter Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain
+ * a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.twitter.storehaus.http
+
+package object compat {
```

Comment:
Of course making the decision to not to support Scala 2.10 any more will make maintenace easier.

I see that #343 just gets approbed so this present PR (#344) does not makes any sense now.

Thank you for upgrading  storehaus to Scala 2.12 and a newer Finagle for the next storehaus version (in our case we don't use Scala 2.10 at our production real servers so it's OK, so the new storehaus is very valuable to us, as always, thanks for that).